onnxruntime/core/providers/js/js_execution_provider.cc

```diff
@@ -168,22 +168,42 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 1
 
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 10, ThresholdedRelu);
 
-class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kJsExecutionProvider, kOnnxDomain, 7, 12, Add);
```

Comment:
I think we don't need
`ONNX_OPERATOR_VERSIONED_TYPED_KERNEL_CLASS_NAME`
to replace
`ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME` here because we don't really need a template class implementation. register in binary.cc should be enough.

Reply:
Done

Reply:
I guess my comments above may be confusing. I mean we can keep them and no need to change them into a template class registration, but I don't mean to remove those lines of `ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME` on binary operators in this file.